packages/fx-core/src/plugins/resource/bot/question.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { MultiSelectQuestion, OptionItem } from "@microsoft/teamsfx-api";
+import { getLocalizedString } from "../../../common/localizeUtils";
+import { QuestionNames } from "./constants";
+import {
+  HostType,
+  HostTypes,
+  NotificationTrigger,
+  NotificationTriggers,
+} from "./resources/strings";
+
+export interface HostTypeTriggerOptionItem extends OptionItem {
+  hostType: HostType;
+  trigger?: NotificationTrigger;
+}
+
+export const FunctionsTimerTriggerOptionItem: HostTypeTriggerOptionItem = optionWithL10n({
+  id: "functionsTimer",
+  hostType: HostTypes.AZURE_FUNCTIONS,
+  trigger: NotificationTriggers.TIMER,
+});
+
+export const FunctionsHttpTriggerOptionItem: HostTypeTriggerOptionItem = optionWithL10n({
```

Comment:
Nit: what's the meaning of optionWith`L10n`?

Reply:
I think l10n is a quite common abbreviation of localization similar to i18n for internationalization